src/core/smallvec_type.hpp

```diff
@@ -241,6 +258,19 @@ class SmallVector {
 		if (to_move > 0) MemMoveT(this->data + pos, this->data + pos + count, to_move);
 	}
 
+	/**
+	 * Extract an item from this vector, moving the following items to fill the gap.
```

Comment:
This method seems to differ from Erase by not changing the order of the items.
Is this the intention, is it needed?
If yes, the docs should point that out. If not, use Erase :)

Reply:
There is also ErasePreservingOrder.
Maybe an overload of that method?

Reply:
ErasePreservingOrder must be new (... well, newer than this originally stone age old commit :)

Reply:
5 years :)